modules/nf-core/metaeuk/easypredict/main.nf

```diff
@@ -0,0 +1,62 @@
+process METAEUK_EASYPREDICT {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::metaeuk=6.a5d39d9"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/metaeuk:6.a5d39d9--pl5321hf1761c0_2':
+        'biocontainers/metaeuk:6.a5d39d9--pl5321hf1761c0_2' }"
```

Comment:
Hm but e.g. here `quay.io` isnt added, will that mixture work fine? Maybe I am missing something.

Reply:
I guess I need to do a pass through all modules, they've not been updated with these changes yet (as you commented on slack)

Reply:
Done!